packages/good-design/src/stories/apps/magik/Magik.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import MagikWidget from "@magiklabs/react-sdk";
+
+export const MagikWidgetOfframp = {
+  args: {
+    isProdUrl: true,
+    defaultCountry: "NG",
+    enableDebugLogs: false,
+    environment: "staging",
+    partnerContext: {},
+    onClose: () => {},
+    apiKey: "r6eMBtUkT2A_aStNmMAfpaOT8fFs0Bew",
```

Comment:
who's apikey is that? should it stay fixed and exposed?

Reply:
This is GoodDollar's API key - you can put this into an env variable if you want.

It doesn't really matter, as it is exposed in the network request.